gtk/src/rt.rs

```diff
@@ -117,6 +117,12 @@ pub fn init() -> Result<(), glib::BoolError> {
         let argv = ::std::env::args().take(1).collect::<Vec<_>>();
 
         if from_glib(ffi::gtk_init_check(&mut 1, &mut argv.to_glib_none().0)) {
+            let result: bool = from_glib(glib::ffi::g_main_context_acquire(
+                glib::ffi::g_main_context_default(),
```

Comment:
```suggestion
                ptr::null_mut(),
```

Would also work

Reply:
I tried to base that code on what `glib::MainContext::acquire` does. It's true that your version would work, but in my opinion it would be not obvious what the null means in this case. From the signature of the function one could see it's a context, but that alone doesn't say what are the semantics of acquiring a null context. The documentation of the function also doesn't mention it. I had to check the sources in GLib repository to confirm it behaves as you said. So I would prefer to leave it as is or create a named constant for that null.

Reply:
Ok with me to keep it as it is :)